Sources/SwiftSyntax/SyntaxText.swift

```diff
@@ -100,7 +100,10 @@ public struct SyntaxText {
     guard !self.isEmpty && !other.isEmpty else {
       return self.isEmpty && other.isEmpty
     }
-    return (other.baseAddress! <= self.baseAddress! && self.baseAddress! + count <= other.baseAddress! + other.count)
+    let doesOtherIncludeSelfBaseAddress: Bool = other.baseAddress! <= self.baseAddress!
+    let sendEndBound = UnsafePointer<UInt8>(self.baseAddress! + count)
```

Comment:
Did you mean `selfEndBound`?

```suggestion
    let sendEndBound = UnsafePointer<UInt8>(self.baseAddress! + count)
```